[2.2] XML files for web version

marcus-oscarsson opened the pull request:
Hi,

Added web version xml file examples and fixed small issue with sample changer mockup (unload)

Marcus

meguiraun commented:
is it only me? I see `#` character on xml file name, only on these two:
![image](https://user-images.githubusercontent.com/12758327/47005774-c34e6700-d134-11e8-9d70-3df8cafbbb7b.png)

Also, which is the differences between qt and web xml files?

marcus-oscarsson commented:
Hey

Nope its not only you, it was emacs temp files. I've removed them now.

There are some differences between the qt and web xml files for instance the Shapes HardwareObject are not the same, and we added a few attributes to the web version which are not used in the qt version. So its maybe clearer to keep them separate, it also makes it easier to just copy the files as they are.

Marcus
